docs/html/cli/pip_install.rst

```diff
@@ -425,15 +425,15 @@ Examples
       .. code-block:: shell
 
          python -m pip install SomeProject@http://my.package.repo/SomeProject-1.2.3-py33-none-any.whl
-         python -m pip install "SomeProject @ http://my.package.repo/SomeProject-1.2.3-py33-none-any.whl"
+         python -m pip install 'SomeProject @ http://my.package.repo/SomeProject-1.2.3-py33-none-any.whl'
```

Comment:
> I'll switch everything over to single quotes unless someone objects.

@uranusjr If this adds to much extra noise to the PR I can hold off on this for a style PR later.